File: js/findKthLargest.js
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */
 var findKthLargest = function(nums, k) {
    nums.sort((a, b) => b - a);

    return nums[k - 1];
};

// const nums = [3,2,1,5,6,4];
// console.log(findKthLargest(nums, 2)); // 5

const nums = [3,2,3,1,2,4,5,5,6];
console.log(findKthLargest(nums, 4)); // 4

File: python/find_kth_largest.py
from typing import List


class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        nums.sort(reverse=True)
        return nums[k - 1]


res = Solution().findKthLargest([1, 4, 3, 77], 1)
print(res)
